### PR TITLE
TASK: Migrate Code for Neos 3.0 compatibility

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

File: Configuration/Settings.yaml [hunk 1]
```diff
@@ -1,43 +1,35 @@
...
     cacheHeaders:
-      # Default and maximum TTL in seconds
-      defaultSharedMaximumAge: ~
-      # Disable sending headers (useful for staging environments)
-      disabled: FALSE
-    # Port to use for reverse lookup in backend module
-    reverseLookupPort: ~
-    # List of ignored  cache tags to skip when flushing caches by tag, e.g. 'TYPO3.Neos:Document' which is used in 'TYPO3.Neos:Menu' elements
-    ignoredCacheTags: []
-    # Log configuration
```
Comment (aertmann) [inline Configuration/Settings.yaml]: some comments are lost here due to the automated configuration migration, please adjust manually instead to and only change what's actually needed..

**Review by aertmann — CHANGES_REQUESTED** [review]

**Comment by aertmann** [comment]
fixed the travis tests

looks good by reading, thanks for taking care..

just one comment regarding the configuration to be fixed

File: Classes/MOC/Varnish/Aspects/ContentCacheAspect.php [hunk 2]
```diff
@@ -39,9 +39,9 @@ public function registerCreateUncached(JoinPointInterface $joinPoint) {
 		$runtime = ObjectAccess::getProperty($proxy, 'runtime', TRUE);
 
 		if ($evaluateContext['cacheForPathDisabled']) {
-			$mocVarnishIgnoreUncached = $runtime->evaluate($evaluateContext['fusionPath'] . '/__meta/cache/mocVarnishIgnoreUncached');
+			$mocVarnishIgnoreUncached = $runtime->evaluate($evaluateContext['typoScriptPath'] . '/__meta/cache/mocVarnishIgnoreUncached');
 			if ($mocVarnishIgnoreUncached !== TRUE) {
-				$this->logger->log(sprintf('Varnish cache disabled due to uncached path "%s" (can be prevented using "mocVarnishIgnoreUncached")', $evaluateContext['fusionPath']), LOG_DEBUG);
+				$this->logger->log(sprintf('Varnish cache disabled due to uncached path "%s" (can be prevented using "mocVarnishIgnoreUncached")', $evaluateContext['typoScriptPath']), LOG_DEBUG);
```
Comment (aertmann) [inline Classes/MOC/Varnish/Aspects/ContentCacheAspect.php]: whoops, seems like some leftover in the core

Comment (daniellienert) [reply Classes/MOC/Varnish/Aspects/ContentCacheAspect.php]: Yeah, there are a lot of internal leftovers.
So this will break in the future again.

**Comment by daniellienert** [comment]
Thanks for reviewing @aertmann, comments are fixed.

**Comment by daniellienert** [comment]
@aertmann - ping?

**Review by aertmann — APPROVED** [review]

**Pull request merged**